client/landing/jetpack-cloud/sections/scan/main.tsx

```diff
@@ -252,7 +255,7 @@ class ScanPage extends Component< Props > {
 					noticeText={ translate(
 						'Failing to plan is planning to fail. Regular backups ensure that should ' +
 							'the worst happen, you are prepared. Jetpack Backup has you covered.'
-					) }
+					).toString() }
```

Comment:
I didn't know about this! It's very useful for when we need to pass a string instead of a translation object.

Reply:
TIL, thanks for sharing!